Dockerfile

```diff
@@ -11,17 +11,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-ARG TRINO_VERSION=391
+ARG TRINO_VERSION=403
 
 FROM trinodb/trino:${TRINO_VERSION}
 
-ARG CONNECTOR_VERSION=4.3.0-391
+ARG CONNECTOR_VERSION=4.4.0-403
 
 USER root:root
 RUN \
     apt-get update && \
     apt-get -y -q install unzip ca-certificates wget uuid-runtime gettext && \
     export BASE_VERSION=$(echo $CONNECTOR_VERSION | cut -d '-' -f 1) && \
+# CONNECTOR-435 Solution
+# Alternative: wget -q -O /tmp/aerospike.zip "https://download.aerospike.com/artifacts/enterprise/aerospike-trino/$BASE_VERSION/aerospike-trino-$CONNECTOR_VERSION.zip" && \
```

Comment:
@mcoberly2 why not replacing the URL?

Reply:
@reugn  Which url you're referring to? The one on 27 seems to be evolved version of the url on 26.